tests/test_fulltextsearch.py

```diff
@@ -0,0 +1,108 @@
+"""Tests for full_text_search.py"""
+
+import unittest
+import logging
+import datetime
+
+from google.appengine.ext import db
+from google.appengine.ext import testbed
+from google.appengine.api import search
+
+import full_text_search
+import model
+
+TEST_DATETIME = datetime.datetime(2010, 1, 1, 0, 0, 0)
+
+def create_index(person):
+    full_text_search.create_index(
+        record_id=person.record_id,
+        repo=person.repo,
+        given_name=person.given_name,
+        family_name=person.family_name,
+        full_name=person.full_name,
+        alternate_names=person.alternate_names
+    )
+
+class FullTextSearchTests(unittest.TestCase):
+    def setUp(self):
+        self.tb = testbed.Testbed()
+        self.tb.activate()
+        self.tb.init_search_stub()
+        self.p1 = model.Person(
```

Comment:
Optional: It may be slightly cleaner to use model.Person.create_original_with_record_id:
https://github.com/google/personfinder/blob/master/app/model.py#L231
Then you can specify record_id instead of key_name directly.